app/Resources/views/base.html.twig

```diff
@@ -154,6 +154,12 @@
             <script>
                 $(document).ready(function() {
                     hljs.initHighlightingOnLoad();
+
+                    $('[data-toggle="confirmation"][data-message]').on('click', function (event) {
```

Comment:
and btw, using event delegation would be better than doing a DOM traversal on dom ready to find all such elements